gettingstarted/settings.py

```diff
@@ -105,15 +127,41 @@
 
 USE_I18N = True
 
-USE_L10N = True
-
 USE_TZ = True
 
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/2.0/howto/static-files/
 
-STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
+STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
```

Comment:
With Python 3's `Path`, this should now work (and is what the WhiteNoise docs say to do):
https://whitenoise.evans.io/en/stable/django.html

```suggestion
STATIC_ROOT = BASE_DIR / "staticfiles"
```